spec/services/delivery_request_service_spec.rb

```diff
@@ -155,10 +168,18 @@
         end
       end
 
-      it "marks the email as failed" do
-        expect { described_class.call(email: email) }
-          .to change { email.reload.status }
-          .to("failed")
+      it "marks the email as failed and sets sent_at time" do
+        freeze_time do
+          expect { described_class.call(email: email) }
+            .to change { email.status }.to("failed")
```

Comment:
We're missing the actual check for `sent_at` here.

Reply:
That was old text describing the test - have updated it accordingly!